### PR TITLE
fix: restore example site

### PR DESCRIPTION
**Pull request by hgaitanis**
## Description

https://digital-engagement.atlassian.net/browse/MYVIC-116
Added vue router to the example app and bring the original demo page to the app. Use dynamic import to split code by route.

Original PR: https://github.com/dpc-sdp/myvic-components/pull/90

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [x] The PR title is no longer than 64 characters.
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core

### CODE REVIEW

File: examples/vue-example-app/src/components/utils/getData.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import axios from 'axios'
 import sensors from './sensors'
 
-const ABS_DATA_BASE_URL = 'https://proxy-myvic-components-production.lagoon.vicsdp.amazee.io/abs/sdmx-json/data/'
+const ABS_DATA_BASE_URL = 'https://proxy.production.myvic-components.sdp2.sdp.vic.gov.au/abs/sdmx-json/data/'
```
Comment (hgaitanis) [inline examples/vue-example-app/src/components/utils/getData.js:4]: This should work in production but you'll need to disable CORS for it to work properly

File: examples/vue-example-app/src/router/index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: Home
+  },
+  {
+    path: '/demo',
+    name: 'Demo',
+    component: () => import(/* webpackChunkName: "demo" */ '../views/Demo.vue')
```
Comment (rubyllamadora) [inline examples/vue-example-app/src/router/index.js:16]: Is there any reason why Demo and Home are in views directory and not in pages?

File: examples/vue-example-app/src/router/index.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import VueRouter from 'vue-router'
+import Home from '../views/Home.vue'
+
+Vue.use(VueRouter)
+
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: Home
```
Comment (rubyllamadora) [inline examples/vue-example-app/src/router/index.js:11]: Should the Demo be the root route `/`? Do we still need this as Home or should we call this MapLayout.vue?

**Review by rubyllamadora — COMMENTED** [review]
@hgaitanis Demo page looks good. You might want to maintain yarn.lock instead and remove package-lock.json? Also some comments on routing if we've deliberately organised it that way.

File: examples/vue-example-app/src/router/index.js (hunk 3 above)
Comment (hgaitanis) [reply examples/vue-example-app/src/router/index.js:11]: according to the JIRA story, we should preserve the root route as is and place the examples under /demo

File: examples/vue-example-app/src/router/index.js (hunk 2 above)
Comment (hgaitanis) [reply examples/vue-example-app/src/router/index.js:16]: I think it makes sense to move all these in pages, I will do that. I will remove package-lock.json as well.

File: examples/vue-example-app/src/router/index.js (hunk 3 above)
Comment (rubyllamadora) [reply examples/vue-example-app/src/router/index.js:11]: makes sense.

**Review by rubyllamadora — APPROVED** [review]

**Pull request merged**